### ISSUE TITLE
Temple warning

### ISSUE THREAD

**Issue by atompkins** [fallenswordhelper/fallenswordhelper#27]
Could we put miniature buttons on the warning label?

**Comment by atompkins** [comment]
I've got this coded in DEV. Looks really nice. Very Ajax.
I seem to recall a conversation with HCS many moons ago that they didn't want this though...

**Comment by atompkins** [comment]
c4b2b4086250bbd55717a5f921c0c7e0db3ee502
